sklearn/base.py

```diff
@@ -519,7 +520,7 @@ def score(self, X, y=None):
 
 
 class OutlierMixin:
-    """Mixin class for all outlier detection estimators in scikit-learn."""
+    """Mixin class for all outlier rejection estimators in scikit-learn."""
```

Comment:
I am not sure to understand why we need to use rejection instead of detection here. An estimator inheriting from OutlierMixin exactly corresponds to an outlier detector, not necessarily an outlier rejector used in a pipeline.

Reply:
ups, this should be detection. The rejection is the new OutlierRejectionMixin. Good catch.